gtsfm/bundle/bundle_adjustment.py

```diff
@@ -349,16 +352,6 @@ def run_ba(
             Valid mask as a list of booleans, indicating for each input track whether it was below the re-projection
                 threshold.
         """
-        logger.info(
-            "Input: %d tracks on %d cameras", initial_data.number_tracks(), len(initial_data.get_valid_camera_indices())
-        )
-        if initial_data.number_tracks() == 0 or len(initial_data.get_valid_camera_indices()) == 0:
```

Comment:
just curious, why remove this?

Reply:
@akshay-krishnan

Reply:
wanted to print it for each stage, forgot to paste above. did it now.